src/api/hocrrenderer.cpp

```diff
@@ -342,6 +345,31 @@ char* TessBaseAPI::GetHOCRText(ETEXT_DESC* monitor, int page_number) {
           tcnt++;
         }
       }
+    } else if (tesseract_->lstm_choice_mode == 4) {
+      for (auto timestep : *CTCMap) {
+        if (timestep.size() > 0) {
+          hocr_str << "\n       <span class='ocrx_cinfo'"
+                   << " id='"
+                   << "lstm_choices_" << page_id << "_" << wcnt << "_" << tcnt
+                   << "'>";
+          for (auto& j : timestep) {
+            float conf = 100 - tesseract_->lstm_rating_coefficient * j.second;
```

Comment:
The confidence values are closer to the values above (`res_it->Confidence(RIL_SYMBOL)`) without the factor `tesseract_->lstm_rating_coefficient`. This also fixes the large variation when the rating coefficient is changed to a different value. @noahmetzger, please check.